tests/Button/Button.spec.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from "react";
+import { shallow } from "enzyme";
+import  {Button} from "../../src/components/Button"
+
+describe("Tests for button component",() =>{
+  let setup = (value) => {
+    const props ={
+      isDisabled: value,
+      onClick: () => {},
+      icon: <button></button>,
+      label : "",
+			title : "",
```

Comment:
Indentation is a bit off here too

Reply:
I Don't know why its happening as in my editor it looks good

Reply:
![screenshot from 2018-02-04 16-12-14](https://user-images.githubusercontent.com/22280901/35776683-0e7b4774-09c7-11e8-9ddd-d13c5bab8262.png)

Reply:
Did you try running tslint?

Also, please rebase once, so that travis can test